app/models/user.rb

```diff
@@ -41,5 +40,26 @@ def set_default_role
   has_one :profile
   has_many :application_letters
   has_many :requests
-       
+
+  def participation_count
+    ApplicationLetter.where(:user_id => id).count()
+  end
+
+  def accepted_application_count
+    ApplicationLetter.where(:user_id => id, :status => true).count()
+  end
+
+  def rejected_application_count
+    ApplicationLetter.where(:user_id => id, :status => false).count()
+  end
+
+  def self.human_attribute_name(*args)
+    case args[0].to_s
```

Comment:
Was planed to be more extensive. Will be updated with all the localization changes